src/main/resources/org/sagebionetworks/web/client/widget/docker/modal/AddDockerCommitModalViewImpl.ui.xml

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+    xmlns:g="urn:import:com.google.gwt.user.client.ui"
+    xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+    xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+
+  <b:Modal ui:field="addDockerCommitModal">
+    <b:ModalBody>
+      <bh:Div ui:field="synAlertContainer"/>
+      
+      <b:FieldSet>
+        <b:FormGroup>
+          <b:FormLabel for="digestTextBox">Docker Image Digest *</b:FormLabel>
```

Comment:
I like the '*' indicator for required, but I don't think we have this across the app, so I would not include in this one instance